fe/fe-core/src/test/java/com/starrocks/sql/plan/ExpressionTest.java

```diff
@@ -1740,4 +1740,98 @@ public void testCastStringDouble() throws Exception {
             connectContext.getSessionVariable().setCboEqBaseType("VARCHAR");
         }
     }
+
+    public void testTypeofFunctionPlan() throws Exception {
```

Comment:
Can you move these test cases to SQL tests?

Reply:
> Can you move these test cases to SQL tests?

done